### DISCUSSION TITLE
Help For new HTML / CSS And JavaScript Coder

### DISCUSSION THREAD

**Discussion by maligaurav947** [JdeRobot/RoboticsAcademy#1519] [Q&A] [answered]
Hey Coder,
I Always Look Forward to the new Language Learning Website
if you have any related To HTML, CSS, JAVASCRIPT. Please Let Me Know That Much appreciate

This is My Collection Of Website
-------------------------------------------------------------------------------------
https://www.tutorialspoint.com/html/index.htm
https://www.w3schools.com/html/
https://www.freecodecamp.org/news/html-and-html5-example/
https://html.com/

**Comment by pariaspe** [comment, marked as answer]
Hi @maligaurav947,
Robotics-Academy is an open source collection of exercises and challenges to learn **robotics** and **computer vision** in a practical way. 
By now, it does not have exercises on the topics you are looking for, but you can find exercises about drone programming, about mobile robots, about autonomous cars...
